README.md

```diff
@@ -93,72 +73,156 @@ npm install
 
 ## Usage
 
-### Generating a new key pair
+### Installing Support for Key Types
 
-Ed25519:
+In order to use this library, you will need to import and install driver
+libraries for key types you'll be working with via the `use()` method.
+
+For example, to use this library with only the `ed25519` key type:
 
 ```js
-const {Ed25519KeyPair} = require('crypto-ld');
+import {CryptoLD} from 'crypto-ld';
+const cryptoLd = new CryptoLD();
+
+cryptoLd.use(require('ed25519-key-pair'));
```

Comment:
Please update to pull in the module via the ES6 Module syntax.

Reply:
Fixed, thanks.